scripts/encrypt.py

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env python
+""" Test encryption
```

Comment:
Is this script just for testing the bcrypt interface? If so, do we need it committed?